### DISCUSSION TITLE
Orphaned configuration

### DISCUSSION THREAD

**Discussion by pagabee** [grblHAL/core#12] [General]
Hello,

I try to compile the project and there are some problems:
- Orphaned configuration. No base extension cfg exists for com.st.stm32cube.ide.mcu.gnu.managedbuild.config.exe.release.1170194714.610601776
- Nothing to build for project GRBL Driver STM32F401
- It is not possible to modify the symbols or add new one.

![orp](https://user-images.githubusercontent.com/82469736/114607766-35b84780-9c9d-11eb-90d5-ceb4780e9be5.png)

![ver](https://user-images.githubusercontent.com/82469736/114608462-00f8c000-9c9e-11eb-9140-d127a1f27c9c.jpg)

What is the problem?

**Comment by terjeio** [comment]
Incompatible IDE? I am using the [STM32CubeIDE to build](https://github.com/grblHAL/STM32F4xx#readme).

You may try to set up a new configuration from scratch via _Manage Configurations..._ or delete the project files and importing the source into a new project - but it is perhaps easier to use the STM32CubeIDE?

**Comment by pagabee** [comment]
It works with the STM32CubeIDE.
Thanks....
